.github/workflows/acceptance.yml

```diff
@@ -415,9 +682,83 @@ jobs:
           name: cypress-videos
           path: cypress/videos
 
-  guillotina:
+  # guillotina:
```

Comment:
@sneridagh Is it intended to disable the guillotina tests?

Reply:
Undecided on this one, thinking in remove them, since there was no activity on that side for years. It will be easy to re-enable them, maybe postpone the decision for later.

Reply:
@sneridagh I don't feel strongly about it, but I would put it in a separate PR so that it shows up a bit more clearly in the release notes.